homeassistant/components/insteon/api/insteon_dev_view.py

```diff
@@ -0,0 +1,36 @@
+"""Frontend view for Insteon development."""
+
+from aiohttp import web
+
+from homeassistant.components.http import HomeAssistantView
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+
+class InsteonFrontendDev(HomeAssistantView):
+    """Dev View Class for Insteon."""
+
+    requested_file = None
+    url = r"/insteonfiles/{requested_file:.+}"
+
+    def __init__(self, hass, frontend_url: str) -> None:
+        """Init the InsteonFrontendDev class."""
+
+        self._session = async_create_clientsession(hass, verify_ssl=False)
+        self.requires_auth = False
+        self.name = "insteon_files:frontend"
+        if frontend_url.endswith("/"):
+            frontend_url = frontend_url[:-1]
+        self._url_base = frontend_url
+
+    async def get(self, request, requested_file):
+        """Handle Insteon Web requests."""
+        self.requested_file = requested_file
+        requested = requested_file.split("/")[-1]
+        url = f"{self._url_base}/{requested}"
+        request = await self._session.get(url)
```

Comment:
Why proxy the files from another webserver? Can't you serve it statically from disk like we do with HA frontend ?

Reply:
This is just for development purposes. If there is an entry in the `configuration.yaml` as such:
```
insteon:
  dev_url: http://192.168.1.100:500
```
Then the frontend is served from the dev_url rather than the installed module. Got this idea from your code I think.

Reply:
I understand, but why not do like the HA frontend does, and register a static path where the dev server is generating the files on disk.

Reply:
I am sure I could do it that way. I found it easier to do this way to keep the dev containers completely separate but I am fine either way if you feel it is important for consistancy. I would change dev_url to something like frontend_path then.

Reply:
Now I remember. I stole this code from HACS.

Reply:
Yeah I would follow the exact same pattern as HA does for its frontend.